[bmon] Failed to build

```
checking for NcursesW wide-character library... yes
checking for working ncursesw/curses.h... no
checking for working ncursesw.h... no
checking for working ncurses.h... no
configure: WARNING: could not find a working ncursesw/curses.h, ncursesw.h or ncurses.h
configure: error: requires an X/Open-compatible Curses library with color
bmon failed to build: `./configure --prefix=/usr/local --libdir=/usr/local/lib --mandir=/usr/local/share/man --build=armv7l-cros-linux-gnueabihf --host=armv7l-cros-linux-gnueabihf --target=armv7l-cros-linux-gnueabihf --program-prefix='' --program-suffix=''             --without-ncurses             --with-ncursesw` exited with 1
```

--- a/packages/rgb.rb
+++ b/packages/rgb.rb
@@ -2,35 +2,21 @@ require 'package'
 
 class Rgb < Package
   description 'X color name database'
-  homepage 'https://xorg.freedesktop.org'
+  homepage 'https://xorg.freedesktop.org/'
   version '1.0.6'
   license 'custom'
   compatibility 'all'
   source_url 'https://www.x.org/releases/individual/app/rgb-1.0.6.tar.bz2'
   source_sha256 'bbca7c6aa59939b9f6a0fb9fff15dfd62176420ffd4ae30c8d92a6a125fbe6b0'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6_armv7l/rgb-1.0.6-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6_armv7l/rgb-1.0.6-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6_i686/rgb-1.0.6-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6_x86_64/rgb-1.0.6-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '383f3eb09ad356d72aec226b445f8b69f33f64ddc51279e84e046fcec83a293a',
-     armv7l: '383f3eb09ad356d72aec226b445f8b69f33f64ddc51279e84e046fcec83a293a',
-       i686: 'f0cf3ee49517e01bd7d97bf017b48253f19bb6dd304ab4b59f6612db57756fe1',
-     x86_64: '021c3ce30ac996c8bfbcda435612a1d5d0a449f96ff1ae88f0b5c7903b7ca141',
-  })
+  depends_on 'xorg_proto' => :build
 
   def self.build
-    system './autogen.sh'
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}"
+    system "#{CREW_ENV_OPTIONS} ./configure #{CREW_OPTIONS}"
     system 'make'
   end
 
   def self.install
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
 end

--- a/packages/rgb.rb
+++ b/packages/rgb.rb
@@ -9,6 +9,19 @@ class Rgb < Package
   source_url 'https://www.x.org/releases/individual/app/rgb-1.0.6.tar.bz2'
   source_sha256 'bbca7c6aa59939b9f6a0fb9fff15dfd62176420ffd4ae30c8d92a6a125fbe6b0'
 
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6-1_armv7l/rgb-1.0.6-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6-1_armv7l/rgb-1.0.6-1-chromeos-armv7l.tar.xz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6-1_i686/rgb-1.0.6-1-chromeos-i686.tar.xz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rgb/1.0.6-1_x86_64/rgb-1.0.6-1-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '27166ea52928b8accc832ec1487f7a93cd25db22920a990d81c612a4b32ebc96',
+     armv7l: '27166ea52928b8accc832ec1487f7a93cd25db22920a990d81c612a4b32ebc96',
+       i686: '4af3c2ad090fc4a3bb80e574338452b26d6893e2f2185ce8bdc6f6ce73b2824d',
+     x86_64: '5921f6f8cf98c4a7ff6760921a4a3b3092bb875caf3baea4b5f4e515d837edcc'
+  })
+
   depends_on 'xorg_proto' => :build
 
   def self.build

--- a/packages/rgb.rb
+++ b/packages/rgb.rb
@@ -12,7 +12,7 @@ class Rgb < Package
   depends_on 'xorg_proto' => :build
 
   def self.build
-    system '[ -x configure ] || ./autogen.sh'
+    system '[ -x configure ] || NOCONFIGURE=1 ./autogen.sh'
     system "#{CREW_ENV_OPTIONS} ./configure #{CREW_OPTIONS}"
     system 'make'
   end

--- a/packages/rgb.rb
+++ b/packages/rgb.rb
@@ -3,7 +3,7 @@ require 'package'
 class Rgb < Package
   description 'X color name database'
   homepage 'https://xorg.freedesktop.org/'
-  version '1.0.6'
+  version '1.0.6-1'
   license 'custom'
   compatibility 'all'
   source_url 'https://www.x.org/releases/individual/app/rgb-1.0.6.tar.bz2'
@@ -12,6 +12,7 @@ class Rgb < Package
   depends_on 'xorg_proto' => :build
 
   def self.build
+    system '[ -x configure ] || ./autogen.sh'
     system "#{CREW_ENV_OPTIONS} ./configure #{CREW_OPTIONS}"
     system 'make'
   end

